Main.html

```diff
@@ -58,8 +60,16 @@ <h3>Личные качества:</h3>
     </div>
   </div>
 
-  <div class="container_2">
-    <h1 id="cont">Контакты</h1>
-    >
+  <div class="container_2" id="cont">
+    <h1>Контакты</h1>
+    <h3>Мои контакты</h3>
+    <ul >
+        <li><img src = "logo/whatsapp.png"><p class="logo_text">+7-951-644-**-**</p></li>
```

Comment:
User **a** element for your phone
https://css-tricks.com/the-current-state-of-telephone-links/

Reply:
and don't be afraid about exposing it 🙂

Reply:
fixed